infrap4d/CMakeLists.txt

```diff
@@ -19,26 +16,8 @@ target_include_directories(infrap4d PRIVATE ${STRATUM_SOURCE_DIR})
 
 target_link_libraries(infrap4d PRIVATE
     -Wl,--whole-archive
-    stratum
+    stratum_hal_lib_tdi
```

Comment:
This is unacceptable. I need the bulk of stratum as a _static_ library, not a _shared_ library.

Reply:
stratum can now be configured to build static or shared libs. Built by default as static.